services/frontend/src/store/sagas/profiles.sagas.js

```diff
@@ -13,8 +13,34 @@ function* profilesFetchAll() {
     yield put(profileSkillsSlice.actions.addProfileSkills(profileSkills));
 }
 
+function* profilesPatchPersonalDetails({payload}, success) {
+    try {
+        const result = yield call(api.service("profiles").patch, payload.id, payload);
+        const normalizedProfile = Profile.normalizeProfile(result);
+
+        yield put(profilesSlice.actions.setProfile(normalizedProfile));
+    }
+    catch (error) {
+        yield put(notificationRequestsSlice.setNotification.actions.request(
```

Comment:
What's the advantage here of having a whole request slice for something that doesn't require the request lifecycle when you could... just fire the `notificationsSlice.actions.setNotification` directly?

Reply:
Not really any bonus it seems, I've swapped over to using the normal slice.